launch/launch/actions/execute_process.py

```diff
@@ -86,6 +86,7 @@ def __init__(
         prefix: Optional[SomeSubstitutionsType] = None,
         output: Optional[Text] = None,
         log_cmd: bool = False,
+        required: bool = False,
```

Comment:
I made a comment about the approach here: https://github.com/ros2/launch/issues/174#issuecomment-460747142

I don't think it is necessarily a blocker, but it might be worth thinking about before merging this.